### PR TITLE
JBPM-8076 - Add Okta workitem to workitem repository

### PR DESCRIPTION
**Pull request by tsurdilo**

### CODE REVIEW

File: okta-workitem/pom.xml [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jbpm</groupId>
+      <artifactId>jbpm-workitems-core</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jbpm</groupId>
+      <artifactId>jbpm-document</artifactId>
```
Comment (mswiderski) [inline okta-workitem/pom.xml]: this does not seem to be used at all in these work item handlers

Comment (tsurdilo) [reply okta-workitem/pom.xml]: updated - thanks!

**Comment by tsurdilo** [comment]
Jenkins retest please

**Comment by tsurdilo** [comment]
build seems stuck :)

**Comment by tsurdilo** [comment]
rebased, updated repo integration tests

**Pull request merged**